draft-privacypass-rate-limit-tokens.md

```diff
@@ -1125,6 +1225,23 @@ anon_issuer_origin_id = HKDF-Hash(secret=index_result,
 
 This section describes security considerations relevant to the use of this protocol.
 
+## Client Secret Use
+
+The Client Secret key is used for two purposes in this protocol: (1) computing request
+signatures and (2) computing the Anonymous Issuer Origin ID (with the corresponding
+public key). In general, using the same cryptographic key for two distinct purposes
+is considered bad practice. However, analysis of this protocol demonstrates that it
+is safe in this context. The Client Secret MUST NOT be used for any purpose outside
```

Comment:
Beyond the fact that it is safe, can we explain why it needs to be reused here?

Reply:
I gave the reasons for its reuse in the first sentence, so I think is sufficient (without explaining all the previous failed attempts at this protocol).

Reply:
The first sentence about the client secret use doesn't say why the request signatures and Anonymous Issuer Origin ID are related. Can we add just a couple words on that?